0000-route-broadcasting-protocol/RouteBroadcastingProtocol.asn

```diff
@@ -0,0 +1,60 @@
+RouteBroadcastingProtocol DEFINITIONS AUTOMATIC TAGS ::= BEGIN
+
+IMPORTS
+  UInt8,
+  UInt16,
+  UInt32
+  FROM GenericTypes
+  Address
+  FROM InterledgerTypes
+;
+
+-- Route Control Request
+
+SyncMode ::= INTEGER {
+  idle (0),
+  sync (1)
+} (0..255)
+
+RoutingTableId ::= OCTET STRING (SIZE (16))
+
+Feature ::= OCTET STRING
+
+RouteControlRequestData ::= SEQUENCE {
+  syncMode SyncMode, -- the state that the node wants the counterpart node to be
+  routingTableId RoutingTableId, -- the routing table ID that the requesting node knows at that point
+  knownEpochIndex UInt32, -- the epoch that the requesting node knows at that point
+  features SEQUENCE OF Feature -- reserved for the future, currently not used
+}
+
+-- Route Update Request
+
+Auth ::= OCTET STRING (SIZE (32))
+
+Property ::= SEQUENCE {
+  meta UInt8,
```

Comment:
It might be worth defining the meaning of `meta`. See [ilp-protocol-ccp](https://github.com/interledgerjs/ilp-protocol-ccp/blob/master/src/index.ts#L272) for some examples of the data encoded into this byte field.

From what I can tell, the following fields exist: `optional`, `transitive`, `partial`, `isUtf8`.

Reply:
I actually knew the options but I hesitated to write it because it seemed that the options were not used. Should I write it nevertheless?

Reply:
I think it would be fine if these definitions were written later (e.g., after merging this PR), although the terms are used in the source-code (though undocumented), and so it would be good to outline them in the event some implementation wanted to start using them.